assets/js/aplus.js

```diff
@@ -606,3 +606,33 @@ $.fn.highlightCode = function(options) {
         });
     };
 })(jQuery, window, document);
+
+
+/**
+ * prevent double submit of forms
+ */
+
+$(function() {
+
+    $.fn.preventDoubleSubmission = function() {
+  $(this).on('submit',function(e){
+    var $form = $(this);
+
+    if ($form.data('submitted') === true) {
+      // Previously submitted - don't submit again
+      e.preventDefault();
+    } else {
+      // Mark it so that the next submit can be ignored
+      $form.data('submitted', true);
+      //disable the button
+      $form.find('[type="submit"]').prop('disabled', 'disabled')
+    }
+  });
+
+  // Keep chainability
+  return this;
+};
+//prevent double submisssion of exercise form
+$('#exercise-page-content').preventDoubleSubmission();
```

Comment:
This doesn't match to the form. #exercise-page-content form should do it.

Reply:
This also needs to be inside `$(function() {...})` so it's not executed before page is fully loaded.